chore(examples): Updating vendor examples (#3966)
kubeshop/tracetest: docs/docs/examples-tutorials/recipes/running-tracetest-with-lightstep.mdx

## Prerequisites

**Tracetest Account**:

- Sign up to [`app.tracetest.io`](https://app.tracetest.io) or follow the [get started](/getting-started/installation) docs.
- Have access to the environment's [agent API key](https://app.tracetest.io/retrieve-token).

**Docker**: Have [Docker](https://docs.docker.com/get-docker/) and [Docker Compose](https://docs.docker.com/compose/install/) installed on your machine.

## Run This Quckstart Example

The example below is provided as part of the Tracetest project. You can download and run the example by following these steps:

Clone the Tracetest project and go to the example folder:

```bash
git clone https://github.com/kubeshop/tracetest
cd tracetest/examples/tracetest-lightstep
```

Follow these instructions to run the quick start:

1. Copy the `.env.template` file to `.env`.
2. Fill out the [TRACETEST_TOKEN and ENVIRONMENT_ID](https://app.tracetest.io/retrieve-token) details by editing your `.env` file.
3. Fill out the [LIGHTSTEP_ACCESS_TOKEN](https://docs.lightstep.com/otel/otel-quick-start) details by editing your `.env` file.
4. Run `docker compose run tracetest-run`.
5. Follow the links in the output to view the test results.

> Follow along with the sections below for an in detail breakdown of what the example you just ran did and how it works.

Reviewer: ```suggestion
Follow the sections below for a detailed breakdown of what the example you just ran did and how it works.
```

Let's fix this by configuring Tracetest and OpenTelemetry Collector to forward trace data to both Lightstep and Tracetest.

**How to Send Traces to Tracetest and Lightstep**

> The `collector.config.yaml` explains that. It receives traces via either `grpc` or `http`. Then, exports them to Tracetest's OTLP endpoint `tracetest:4317` in one pipeline, and to Lightstep in another.

Reviewer 2: ```suggestion
The `collector.config.yaml` explains that. It receives traces via either `grpc` or `http`. Then, exports them to Tracetest Agent's OTLP endpoint `tracetest-agent:4317` in one pipeline, and to Lightstep in another.
```

    traces/lightstep:
      receivers: [otlp]
      processors: [batch]
      exporters: [otlp/lightstep]
```

> ## Running the tests

Reviewer: ```suggestion
## Running the Tests
```
